app/serializers/api/v1/participant_from_induction_record_serializer.rb

```diff
@@ -116,7 +116,10 @@ def eligible_for_funding?(profile)
       end
 
       attribute :updated_at do |induction_record|
-        induction_record.participant_profile.user.updated_at.rfc3339
+        [
```

Comment:
I think we would want to update the `updated_since` flag too, otherwise LPs will be only querying against the user `updated_at`

Reply:
Good point, I can add that. I'm hoping an extra `or` query won't cost too much